sdk/contentsafety/azure-ai-contentsafety/tsp-location.yaml

```diff
@@ -1,3 +1,5 @@
+commit: d1d9db338f7185489525b386538615283b881a37
```

Comment:
We only permit release public SDK which is generated from typespec of `main` branch. Does your SDK follow the rule?

Reply:
Sure, I will update the commit after the spec pr merged